.github/workflows/build-pr.yml

```diff
@@ -12,6 +12,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Clean-up
+        run:  sudo pkill -f geth; sudo pkill -f obscuro; sudo docker stop $(sudo docker ps -a -q); sudo docker rm $(sudo docker ps -a -q)
```

Comment:
can you make that `obscuronode` instead of `obscuro`, the github runner has obscuro in the path

Reply:
Not sure `obscuronode` will work. Let me just remove that for now, we can iterate.